src/docs/user/references/Variables.adoc

```diff
@@ -286,6 +286,13 @@ _Some characters of the regular expression must be encoded according to the link
 
 _Some characters of the regular expression must be encoded according to the link:https://en.wikipedia.org/wiki/URL_encoding[specification]._
 |?name=token.* => PA:LIST(,token1,token2)
+
+|Tokens Filtered by Node Source
+|All tokens registered in the provided Node Source. This filter will return a list which contains tokens defined globally on the Node Source and on any Node which belongs to the Node Source. See <<_node_access_token>>.
```

Comment:
Just a suggestion: 
All tokens registered in **a considered** Node Source. This filter **returns** a list which contains: (i) **the** tokens defined globally **at the level of** the Node Source and (ii) **those defined at the level of the Nodes belonging** to the considered Node Source.